examples/passport-login/src/__tests__/acceptance/passport-login.acceptance.ts

```diff
@@ -40,6 +40,8 @@ describe('example-passport-login acceptance test', () => {
    */
   before(() => MockTestOauth2SocialApp.startMock());
   after(MockTestOauth2SocialApp.stopMock);
+
+  // eslint-disable-next-line mocha/handle-done-callback
   before(async function setupApplication(this: Mocha.Context) {
```

Comment:
This is a false alarm from `mocha` plugin. The rule should only apply to `done` instead of `this`.

Reply:
Can you please open a bug report for the `mocha` plugin or perhaps send a pull request to fix the problem?

Reply:
https://github.com/lo1tuma/eslint-plugin-mocha/issues/270